src/Header/Header.js

```diff
@@ -0,0 +1,51 @@
+import { useState, useEffect } from "react";
+import "./Header.css";
+import logoImg from "../assets/logo.svg";
+
+function Header() {
+  let [userInfo, setUserInfo] = useState(); //로그인정보 유무로 각각 로그인버튼/프로필정보 출력
+  const invisible = { display: "none" };
+  const visible = { display: "inherit" };
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const response = await fetch(
+        "https://bootcamp-api.codeit.kr/api/sample/user"
```

Comment:
이런 API 고유 주소의 경우는 상수화 해서 한 곳에 보관해 import를 받아 사용하는 것을 추천드립니다.
상수화의 주요 목적은 휴먼 에러(오타) 방지를 위함입니다~
```suggestion
    const API_BASE_URL = 'https://bootcamp-api.codeit.kr'; // 이건 상수 파일로 따로 뺏다고 가정

        `${API_BASE_URL}/api/sample/user`
```